docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -0,0 +1,419 @@
+2. Navigate to the [dashboard](https://liveblocks.io/dashboard) and click "Add
+   project..."
+3. Create a new project with "Development" as the environment.
+4. Go to "API keys" on the side menu, reveal your secret key, and copy it
+5. Create a file called `.env.local` alongside `package.json` in your project.
+6. Place your secret key into the file as `LIVEBLOCKS_SECRET_KEY`.
+
+Liveblocks will now work! Your `.env.local` file should look similar to this:
+
+```
+LIVEBLOCKS_SECRET_KEY=sk_dev_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+```
+
+## Authentication
+
+The Liveblocks starter kit uses [NextAuth.js](https://next-auth.js.org/) for
+authentication, meaning tens of different authentication providers can be used
+with your app. A demo authentication system is used by default, but it's easy to
+add real providers, such as GitHub, Google, Auth0.
+
+<details>
+  <summary>How to set up GitHub authentication</summary>
+
+### GitHub authentication
+
+It's possible to allow users to sign up to your app using GitHub. This is how to
+implement it.
+
+#### Getting the secrets
+
+1. Go to [Developer Settings](https://github.com/settings/apps) on GitHub and
+   click "New GitHub App"
+2. Enter an app name (e.g. `Liveblocks Starter Kit (dev)`). You'll need a new
+   app for each environment, so it's helpful to place "dev" in the name.
+3. Add a homepage URL—this isn't important now, so a placeholder will do.
+4. Find the "Callback URL" input just below, and add your local development URL
+   (e.g. `http://localhost:3000`)
+5. Look for the "Webhook" section and make sure to uncheck "Active".
+6. Use the remaining default settings and press "Create GitHub App"
+7. On the next page under "Client secrets", press "Generate a new client
+   secret".
+8. Copy this secret into the `/.env.local` file as `GITHUB_SECRET`
+9. Go back to the previous page and find the "Client ID" near the top. Copy this
+   into your `.env.local` file as `GITHUB_ID`
+
+Almost there! `.env.local` should now look similar to this:
+
+```
+LIVEBLOCKS_SECRET_KEY=sk_dev_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+GITHUB_SECRET=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+GITHUB_ID=XXXXXXXXXXXXXXXXXXXX
+```
+
+#### Add the provider
+
+Next, go to `/pages/api/auth/[...nextAuth].ts` in your project, and add a GitHub
+provider:
+
+```ts
+import GithubProvider from "next-auth/providers/github";
+
+export const authOptions = {
+  // ...
+  providers: {
+    GithubProvider({
+      clientId: process.env.GITHUB_CLIENT_ID as string,
+      clientSecret: process.env.GITHUB_CLIENT_SECRET as string,
+    }),
+  }
+};
+```
+
+And remove `CredentialsProvider` from the same object, as this was only necessary for the demo authentication system:
+```ts
+export const authOptions = {
+  // ...
+  providers: {
+    // Delete from here
+    CredentialsProvider({
+      // ...
+    }),
+    // Until here
+  }
+};
+```
```

Comment:
```suggestion
```

The installer will now do this for you, so we can remove this section. Though we could consider keeping it in case people don't use the installer? What do you think? @taylorpwhipp